src/pages/es/guides/troubleshooting.mdx

```diff
@@ -139,6 +140,12 @@ nodeLinker: "node-modules"
 
 En Astro, usar una etiqueta `<head>` funciona de igual manera que cualquier otra etiqueta HTML: no es movida a la parte superior de la página ni se fusiona con la etiqueta `<head>` existente. Por este motivo es que seguramente desees incluir una sola etiqueta `<head>` por página. Recomendamos escribir esta única etiqueta `<head>` y su contenido en un [componente de plantilla](/es/core-concepts/layouts/).
 
+### An unexpected `<script>` or `<style>` is included
+
+Quizás observes que se importan las etiquetas `<script>` o `<style>` de un componente en tu código HTML incluso si dicho componente no aparece en el resultado final. Esto puede suceder, por ejemplo, cuando [componentes renderizados condicionalmente](/es/core-concepts/astro-components.mdx#html-din%C3%A1mico) no son mostrados. 
```

Comment:
Thanks for the PR @raul-repos!

One link here seems to be broken:

```suggestion
Quizás observes que se importan las etiquetas `<script>` o `<style>` de un componente en tu código HTML incluso si dicho componente no aparece en el resultado final. Esto puede suceder, por ejemplo, cuando [componentes renderizados condicionalmente](/es/core-concepts/astro-components/#html-dinámico) no son mostrados. 
```

Reply:
Fixed, thanks for the review!